pycarol/organization.py

```diff
@@ -0,0 +1,48 @@
+
+
+class Organization:
+
+    def __init__(self, carol):
+        self.carol = carol
+
+    def get_organization_info(self, organization):
```

Comment:
If this is a class... where are the properties for an Organization object? 
Or
this is a helper module... so... no need to add those functions as class methods...

Reply:
I might need this to implement the switch for organizations. 
Still trying to solve this